modules/virtualisation/podman/podman.nix

```diff
@@ -0,0 +1,165 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.podman;
+  toml = pkgs.formats.toml { };
+  json = pkgs.formats.json { };
+
+  inherit (lib) mkOption types;
+
+  podmanPackage = (pkgs.podman.override { inherit (cfg) extraPackages; });
+
+  # Provides a fake "docker" binary mapping to podman
+  dockerCompat = pkgs.runCommandNoCC
+    "${podmanPackage.pname}-docker-compat-${podmanPackage.version}" {
+      outputs = [ "out" "man" ];
+      inherit (podmanPackage) meta;
+    } ''
+      mkdir -p $out/bin
+      ln -s ${podmanPackage}/bin/podman $out/bin/docker
+
+      mkdir -p $man/share/man/man1
+      for f in ${podmanPackage.man}/share/man/man1/*; do
+        basename=$(basename $f | sed s/podman/docker/g)
+        ln -s $f $man/share/man/man1/$basename
+      done
+    '';
```

Comment:
This seems like it should be a feature of the podman package in Nixpkgs, similar to how the yt-dlp package has a `withAlias` argument. For example,

``` nix
pkgs.podman.override { inherit (cfg) extraPackages enableDockerAlias; }
```

Perhaps it would be possible to add this functionality to the podman package in Nixpkgs?

Reply:
It's a good idea. I'll let somebody else handle that due to a lack of time on my end :innocent: